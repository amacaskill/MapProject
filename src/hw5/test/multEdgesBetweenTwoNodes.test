CreateGraph graph1 

AddNode graph1 n1
AddNode graph1 n2

AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2
AddEdge graph1 n1 n2 e3

ListChildren graph1 n1 