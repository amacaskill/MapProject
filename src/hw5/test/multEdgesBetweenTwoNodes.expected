created graph graph1

added node n1 to graph1
added node n2 to graph1

added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n2 in graph1
added edge e3 from n1 to n2 in graph1

the children of n1 in graph1 are: n2(e1) n2(e2) n2(e3)